genericTemplate/.vscode/tasks.json

```diff
@@ -0,0 +1,19 @@
+{
+    "version": "2.0.0",
+    "tasks": [
+        {
+            "label": "use-release-task-message",
+            "command": "echo  'This template has no debug, use the run-container-torizon-release-<arch> task (TASK RUNNER tab) to build the container and run it on the board'",
```

Comment:
should be good to also have this task returning some error code, so the debug session terminates as soon as possible. If not the debug session will be running until the timeout, I think that this could confuse the user.

Reply:
Found a nicer way to do it than how I was doing